overlord/hookstate/ctlcmd/set.go

```diff
@@ -246,7 +246,8 @@ func setConfdbValues(ctx *hookstate.Context, plugName string, requests map[strin
 		return err
 	}
 
-	if confdbstate.IsConfdbHook(ctx) && !strings.HasPrefix(ctx.HookName(), "change-view-") {
+	if confdbstate.IsConfdbHook(ctx) && !strings.HasPrefix(ctx.HookName(), "change-view-") &&
```

Comment:
This could be converted to a list of allowed **modify hooks** or a helper (e.g. `validateModifyHook`)